casepro/cases/tests.py

```diff
@@ -900,6 +904,7 @@ def test_search(self):
             {
                 'id': self.case.pk,
                 'assignee': {'id': self.moh.pk, 'name': "MOH"},
+                'user_assignee': {'id': self.user1.pk, 'name': "Evan"},
                 'contact': {'id': self.ann.pk, 'name': "Ann"},
                 'labels': [{'id': self.aids.pk, 'name': "AIDS"}],
                 'summary': "Summary",
```

Comment:
It gets covered in the `msgs` tests (because that's where the `not full` case is used). I think we should change one of the `msgs` tests to check when there's actually a `user_assignee`.